### PR TITLE
UML-2830-Actor Side Severance Warning

### PR DESCRIPTION
**Pull request by Lbagg1**
# Purpose

_Briefly describe the purpose of the change, and/or link to the JIRA ticket for context_

Fixes UML-2830

## Approach

_Explain how your code addresses the purpose of the change_

## Learning

_Any tips and tricks, blog posts or tools which helped you. Plus anything notable you've discovered about the Use a Lasting Power of Attorney service_

## Checklist

* [ ] I have performed a self-review of my own code
* [ ]~ I have added relevant logging with appropriate levels to my code ~
* [ ] ~New event_codes have been documented on the [wiki page](https://opgtransform.atlassian.net/wiki/spaces/LSML2/pages/3277881441/Understanding+the+event+logs)~
* [ ]~ I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant~
* [ ] I have added tests to prove my work
* [ ] I have added welsh translation tags and updated translation files
* [ ] I have run an accessibility tool on any pages I have made changes to and fixed any issues found
* [ ] I have notified the Interaction Designer of any content changes so that appropriate screenshots/flow diagram changes can be made
* [ ] The product team have tested these changes

### CODE REVIEW

File: service-front/app/test/CommonTest/Service/Lpa/SiriusLpaFactoryTest.php [hunk 1]
```diff
@@ -65,7 +65,16 @@ public function testCanCreateEmptyLpa()
         $this->assertInstanceOf(Lpa::class, $lpa);
         $this->assertEquals('1234', $lpa->getUId());
     }
+    public function testLpaHasSeveranceWarningReturned()
```
Comment (cooperaj) [inline service-front/app/test/CommonTest/Service/Lpa/SiriusLpaFactoryTest.php:68]:
```suggestion

    public function testLpaHasSeveranceWarningReturned(): void
```

File: service-front/app/test/CommonTest/Service/Lpa/SiriusLpaFactoryTest.php [hunk 2]
```diff
@@ -65,7 +65,16 @@ public function testCanCreateEmptyLpa()
...
+    {
+        $factory = new Sirius();
+
+        $lpa = $factory->createLpaFromData(['uId' => '1234','hasSeveranceWarning' => true]);
 
+        $this->assertInstanceOf(Lpa::class, $lpa);
+        $this->assertEquals('1234', $lpa->getUId());
+        $this->assertEquals(true, $lpa->getApplicationHasSeveranceWarning());
+    }
```
Comment (cooperaj) [inline service-front/app/test/CommonTest/Service/Lpa/SiriusLpaFactoryTest.php:77]:
```suggestion
    }
    
```

Whilst you're in this file update all the test methods with a void return as thats good "code scout" behaviour.

File: service-front/app/src/Common/src/Entity/Lpa.php [hunk 3]
```diff
@@ -187,6 +188,17 @@ public function setApplicationHasGuidance(bool $applicationHasGuidance): void
...
+    {
+        return $this->hasSeveranceWarning;
+    }
+
+    public function setApplicationHasSeveranceWarning(bool $hasSeveranceWarning): void
+    {
+        $this->hasSeveranceWarning = $hasSeveranceWarning;
+    }
+
+
```
Comment (cooperaj) [inline service-front/app/src/Common/src/Entity/Lpa.php:201]:
```suggestion
```

**Comment by nickdavis2001** [comment]
This PR is now superseded as this code plus bug-fixes is now in the UML-2766_display_i_and_p_to_viewers branch

**Pull request closed without merging**